label_studio/data_import/functions.py

```diff
@@ -73,10 +74,13 @@ def async_import_background(
             for error in validation_errors:
                 error_message += f'- {error}\n'
 
-            project_import.error = error_message
-            project_import.status = ProjectImport.Status.FAILED
-            project_import.save()
-            return
+            if flag_set('fflag_feat_utc_210_prediction_validation_15082025', user=user):
+                project_import.error = error_message
+                project_import.status = ProjectImport.Status.FAILED
+                project_import.save()
```

Comment:
looks like this change predates the PR, but I think we should use `update_fields` here for sure